### DISCUSSION TITLE
No Cover Art For Non-RP Webradio Streams

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#1553] [Bug Reports]
Latest update has not fixed this. As nobody else is complaining I assume this is due to my settings, but 20231118 does not have this bug.

rAudio 20240107
6.6.9-2 aarch64
Raspberry Pi 3B Rev 1.2
BCM2837 • 891 MiB
4 x Cortex-A53 @ 1.20 GHz

**Comment by hjfisch** [comment]
Same for me . Went back to 20230828 , which runs best for me .

**Comment by cmh714** [comment]
Working for me... http://streaming.live365.com/a40905

**Reply by R101G** [reply]
@cmh714 Thanks, I see the problem now.
 None of my stations in the library have the default rA cover art logo, and unless they do you lose the active cover art.

@rern I think something must have changed around early December?

**Comment by R101G** [comment]
@rern Bug is still present in 20240113.

**Comment by R101G** [comment]
@rern Bug is still present in 20240122.

To reproduce:
Add webradio stream http://streaming.live365.com/a40905 and play it - track cover art is shown on the playback page.
In Library, click on the default rAudio logo and select Change Cover Art. If you change this to another image, only the rAudio logo appears on the playback page.

**Comment by R101G** [comment]
Fixed in 20240127, thanks.
